src/main/java/serhat/workshop_4/Character.java

```diff
@@ -0,0 +1,122 @@
+package serhat.workshop_4;
+
+/**
+ * Created by serhatgemici on 14/02/2017.
+ */
+public class Character {
+    private String nickname;
+    private int level;
+    private int health;
+    private int experience;
+    boolean isAlive;
+    private CharacterClass characterClass;
+    private static final int HEALTH_MAXIMUM = 300;
+
+
+    public static int getHealthMaximum() {
+        return HEALTH_MAXIMUM;
+    }
+
+    public String getNickname() {
+        return nickname;
+    }
+
+    public void setNickname(String nickname) {
+        this.nickname = nickname;
+    }
+
+    public int getLevel() {
+        return level;
+    }
+
+    public void setLevel(int level) {
+        this.level = level;
+    }
+
+    public int getHealth() {
+        return health;
+    }
+
+    public void setHealth(int health) {
+        this.health = health;
+    }
+
+    public int getExperience() {
+        return experience;
+    }
+
+    public void setExperience(int experience) {
+        this.experience = experience;
+    }
+
+    public boolean isAlive() {
+        return isAlive;
+    }
+
+    public void setAlive(boolean alive) {
+        isAlive = alive;
+    }
+
+    public Character(CharacterClass characterClass) {
+        this.characterClass = characterClass;
+    }
+
+    public Character(String nickname, CharacterClass characterClass) {
+
+        this.nickname = nickname;
+        this.characterClass = characterClass;
+        this.level = 0;
+        this.health = HEALTH_MAXIMUM;
+        this.experience = 0;
+        this.isAlive = true;
+    }
+
+    public void healToMaximum() {
+        isAlive = true;
+        health = HEALTH_MAXIMUM;
+    }
+
+    public void getDamage(int damage) {
+        health -= damage;
+        System.out.println("Character " + this.getNickname() + " received damage " + damage +". Health left: " + health);
+
+        if (health < 0) {
+            isAlive = false;
+        }
+    }
+
+    public CharacterClass getCharacterClass() {
+        return characterClass;
+    }
+
+    public void increaseLevel(int level) {
+
+        this.level = level;
+        System.out.println("Congratulations, " + this.nickname + ", you've reached " + this.level + " level!");
+
+    }
+
+    public void increaseExperience(int experiencePoints) {
+        experience += experiencePoints;
+        System.out.println(this.nickname + " got " + experiencePoints + " experience points. Now experience is " + experience + " points \n");
+        if (experience == 0) {
+            this.experience = 0;
+        } else if (experience >= 100 || experience < 200) {
+            this.experience = 1;
```

Comment:
Did you mean this.level? Because I got the following logs:

Fight starts! Alice vs Bob
Character Alice received damage 48. Health left: 252
Character Bob received damage 17. Health left: 283
Character Alice received damage 34. Health left: 218
Character Bob received damage 13. Health left: 270
Character Alice received damage 8. Health left: 210
Character Bob received damage 39. Health left: 231
Character Alice received damage 21. Health left: 189
Character Bob received damage 9. Health left: 222
Character Alice received damage 47. Health left: 142
Character Bob received damage 31. Health left: 191
Character Alice received damage 36. Health left: 106
Character Bob received damage 40. Health left: 151
Character Alice received damage 25. Health left: 81
Character Bob received damage 23. Health left: 128
Character Alice received damage 31. Health left: 50
Character Bob received damage 12. Health left: 116
Character Alice received damage 31. Health left: 19
Character Bob received damage 10. Health left: 106
Character Alice received damage 21. Health left: -2
Character Bob received damage 25. Health left: 81
Alice is dead
Bob got 120 experience points. Now experience is 120 points 

Fight starts! Alice vs Bob
Character Alice received damage 30. Health left: 270
Character Bob received damage 15. Health left: 285
Character Alice received damage 34. Health left: 236
Character Bob received damage 9. Health left: 276
Character Alice received damage 42. Health left: 194
Character Bob received damage 49. Health left: 227
Character Alice received damage 48. Health left: 146
Character Bob received damage 8. Health left: 219
Character Alice received damage 1. Health left: 145
Character Bob received damage 43. Health left: 176
Character Alice received damage 0. Health left: 145
Character Bob received damage 37. Health left: 139
Character Alice received damage 22. Health left: 123
Character Bob received damage 20. Health left: 119
Character Alice received damage 41. Health left: 82
Character Bob received damage 20. Health left: 99
Character Alice received damage 36. Health left: 46
Character Bob received damage 27. Health left: 72
Character Alice received damage 33. Health left: 13
Character Bob received damage 36. Health left: 36
Character Alice received damage 28. Health left: -15
Character Bob received damage 21. Health left: 15
Alice is dead
Bob got 120 experience points. Now experience is 121 points 


Process finished with exit code 0

So, Bob got 240 points in sum, but did not reach the 2nd level